eos-phone-home-if-up

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+[ "$IFACE" != "lo" ] || exit 0
```

Comment:
Yeah that should be fine - my preference would still be that we log whatever the logic is in the code but it's not critical. With that said, I'd still want to see these || cases combined into a single if though.